playground/infrastructure/helper.py

```diff
@@ -134,9 +138,17 @@ async def get_statuses(examples: List[Example]):
     """
     tasks = []
     client = GRPCClient()
-    for example in examples:
-        tasks.append(_update_example_status(example, client))
-    await tqdm.gather(*tasks)
+
+    try:
+        concurrency = int(os.environ["BEAM_CONCURRENCY"])
+        logging.info("override default concurrency: %d", concurrency)
```

Comment:
looks like it happens everytime
what if BEAM_CONCURRENCY not defined?

Reply:
there'll be no limit for the number of concurrent RunCode requests, they'll use memory/cpu proportionally and some will timeout

Reply:
could be handy when running CI/CD on a slow hardware